src/main/scala/com/research/nomad/markii/GUIAnalysis.scala

```diff
@@ -178,17 +178,14 @@ object GUIAnalysis extends IAnalysis {
     }
     for ((eventType, methodInfo) <- viewNode.getInlineClickHandlers) {
       for (act <- ownerActivities) {
-        val method = act.getMethodByNameUnsafe(methodInfo.getName);
+        val method = act.getMethodByNameUnsafe(methodInfo.getName)
         if (method != null) {
-          DynamicCFG.addViewHandlerToEventLoopAct(act, method)
```

Comment:
@LuxiWang99 I removed these myself -- since it turns out to be a very easy thing to do compare to the rest of the diff.